timed/reports/views.py

```diff
@@ -81,10 +81,11 @@ def __init__(self, catch_prefixes, *args, base_qs=None, agg_filters=None, **kwar
     def filter(self, *args, **kwargs):
         if args:  # pragma: no cover
             # This is a check against programming errors, no need to test
-            raise RuntimeError(
+            msg = (
                 "Unable to detect statistics filter type form Q objects. use "
                 "filter_aggregate() or filter_base() instead"
             )
+            raise RuntimeError(msg)
         my_filters = {
```

Comment:
This is a check against programming errors, no need to test 0.0

We could also update the signature to not allow this mistake
```python
    def filter(self, *, **kwargs):
        # no check here cause it's not needed and the simplified signature still enforced a kw only method
        my_filters = {
        ...
```

Reply:
AFAIK this isn't valid python code, maybe you meant this?
```py
def filter(self, /, **kwargs):
  ...
```

Reply:
i've implemented this change here: https://github.com/adfinis/timed-backend/pull/1049/commits/0425fddbdf73e76f1b62e6baac671e263d4e92cb

Reply:
indeed, `*` gobbles args and kwargs while `/` just gobbles args